pkg/credentials/filter/credentials_filter_test.go

```diff
@@ -0,0 +1,77 @@
+package filter
```

Comment:
Never heard that you change the package name for unit tests. Usually its enough to just put _test at the end of the files, so that the build ignores those files.
Also its not possible to open up a second package in a folder right?
If that is not what you meant could you explain this in more detail please.

Reply:
It's possible! Here's [an example](https://github.com/tektoncd/pipeline/blob/db4c05c0294ad33a2f6e7b17cacf53526139616d/pkg/workspace/apply_test.go#L1)

Reply:
I still dont get why you do not want to test non public functions. Unit tests can also be written on non public functions right?

Reply:
Change this and used a filter_test package in all test files of this package